proxy/src/main/java/com/velocitypowered/proxy/config/AnnotatedConfig.java

```diff
@@ -65,76 +62,91 @@ public static Logger getLogger() {
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target({ElementType.FIELD, ElementType.TYPE})
-    public @interface IsMap {
-    }
+    public @interface IsMap {}
 
     /**
      * Indicates that a field is a string converted to byte[]
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target({ElementType.FIELD, ElementType.TYPE})
-    public @interface StringAsBytes {
-    }
+    public @interface StringAsBytes {}
 
     /**
-     * Indicates that a field should be skiped
+     * Indicates that a field should be skipped
      */
     @Retention(RetentionPolicy.RUNTIME)
     @Target({ElementType.FIELD})
-    public @interface Ignore {
-    }
+    public @interface Ignore {}
 
+    /**
+     * Dumps configuration to list of strings
+     * @return configuration dump
+     */
     public List<String> dumpConfig() {
         List<String> lines = new ArrayList<>();
         dumpFields(this, lines);
         return lines;
     }
 
     /**
-     * Dump all field and they annotations to List
+     * Creates TOML configuration from supplied <pre>dumpable</pre>
      *
-     * @param toSave object those we need to dump
+     * @param dumpable object which is going to be dumped
      * @param lines a list where store dumped lines
      */
-    private void dumpFields(Object toSave, List<String> lines) {
-
+    private void dumpFields(Object dumpable, List<String> lines) {
```

Comment:
I personally don't think there was any need to change this method - the style itself was fine. If you're going to do this, I would suggest writing a set of unit tests to verify that things worked before and after.

Reply:
In my opinion this code was too tightly packed together. However yes, unit test is good idea because I noticed one potential logic change in code.